hpat/datatypes/hpat_pandas_series_functions.py

```diff
@@ -469,3 +476,332 @@ def hpat_pandas_series_floordiv_number_impl(lhs, rhs):
+        def hpat_pandas_series_ge_impl(lhs, rhs):
+            """
+            Test:  python -m hpat.runtests hpat.tests.test_series.TestSeries.test_series_op8_integer_scalar
+            Test:  python -m hpat.runtests hpat.tests.test_series.TestSeries.test_series_op8_float_scalar
+            """
+
+            return pandas.Series(lhs._data >= rhs)
+
+        return hpat_pandas_series_ge_impl
+
+    raise TypingError('{} The object must be a pandas.series or scalar. Given rhs: {}'.format(_func_name, rhs))
+
+@overload_method(SeriesType, 'lt')
+def hpat_pandas_series_lt(lhs, rhs):
+    """
+    Pandas Series method 'lt' implementation.
+        https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.Series.lt.html#pandas.Series.lt
+
+    Algorithm: result = lhs.lt(other, level=None, fill_value=None, axis=0)
+
+    Where:
+               lhs: pandas.series
+             other: pandas.series or scalar value
+             level: unsupported
+        fill_value: unsupported
+              axis: unsupported
+            result: pandas.series
+
+    Test:  python -m hpat.runtests hpat.tests.test_series.TestSeries.test_series_op8
+    """
+
+    _func_name = 'Method lt().'
+
+    if not isinstance(lhs, SeriesType):
+        raise TypingError('{} The object must be a pandas.series. Given lhs: {}'.format(_func_name, lhs))
+
+    if isinstance(rhs, SeriesType):
+        def hpat_pandas_series_lt_impl(lhs, rhs):
+            """
+            Test:  python -m hpat.runtests hpat.tests.test_series.TestSeries.test_series_op8
+            """
+
+            return pandas.Series(lhs._data < rhs._data)
+
+        return hpat_pandas_series_lt_impl
+
+    if isinstance(rhs, types.Integer) or isinstance(rhs, types.Float):
+        def hpat_pandas_series_lt_impl(lhs, rhs):
+            """
+            Test:  python -m hpat.runtests hpat.tests.test_series.TestSeries.test_series_op8_integer_scalar
+            Test:  python -m hpat.runtests hpat.tests.test_series.TestSeries.test_series_op8_float_scalar
+            """
+
+            return pandas.Series(lhs._data < rhs)
+
+        return hpat_pandas_series_lt_impl
+
+    raise TypingError('{} The object must be a pandas.series or scalar. Given rhs: {}'.format(_func_name, rhs))
+
+@overload_method(SeriesType, 'gt')
+def hpat_pandas_series_gt(lhs, rhs):
+    """
+    Pandas Series method 'gt' implementation.
+        https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.Series.gt.html#pandas.Series.gt
+
+    Algorithm: result = lhs.gt(other, level=None, fill_value=None, axis=0)
+
+    Where:
+               lhs: pandas.series
+             other: pandas.series or scalar value
+             level: unsupported
+        fill_value: unsupported
+              axis: unsupported
+            result: pandas.series
+
+    Test:  python -m hpat.runtests hpat.tests.test_series.TestSeries.test_series_op8
+    """
+
+    _func_name = 'Method gt().'
+
+    if not isinstance(lhs, SeriesType):
```

Comment:
Original function has many extra arguments `gt(self, other, level=None, fill_value=None, axis=0)`
Could you please implement a checks for them?
I mean is these unsupported parameters are not equals to defaults - throw exception on `Unsupported parameters`
This is not implemented before in this module.

Reply:
@shssf I realized check for unsupported parameters, but it has status of change requested. Should I write different checks for any arguments?

Reply:
@1e-to I see you already implemented checks for all function parameters. Thank you.